### ISSUE TITLE
Design for Adding/ Removing elements from the cdd data table (need by 1/31)

### ISSUE THREAD

**Issue by JanellC** [US-EPA-CAMD/PI-1#2478]
## Context 

Feedback from CAMPD Beta Testing:
Users want the ability to add/ remove columns from the data table before downloading a file

**Comment by AllisonFrancis** [comment]
@JanellC I made the changes we discussed with dev. These wireframes are ready for the discussion on 1/31. Let me know if any changes need to be made prior to.
https://xd.adobe.com/view/4eba0d3a-2cac-4a0f-9299-1b9233af7e2d-1be9/

**Comment by AllisonFrancis** [comment]
@tlageman-epa @JanellC We talked to Anthony and he thinks all of this is fine. He did recommend adding the Apply button, as we don't want changes to be happening while they are still in the modal selection view. That would be flagged as a submission 508 issue. So that button has been added to the ui in addition to the language change we discussed about hide/show to deselect/select. Thanks https://xd.adobe.com/view/4eba0d3a-2cac-4a0f-9299-1b9233af7e2d-1be9/
